sessions/return-oriented-programming/README.md

```diff
@@ -0,0 +1,810 @@
+# Return Oriented Programming
+
+## Recap - ASLR
+ASLR is not the only feature that prevents the compiler and the linker from
+solving some relocations before the binary is actually running. Shared libraries
+can also be combined in different ways, so the first time you actually know the
+address of a shared library is while the loader is running. The ASLR feature is

```

Comment:
I think it should be `the first time you actually know what the address of a shared library is`. Missing `what`.

Reply:
No. Maybe I should rephrase it to "While the loader is running is the first time you actually know address of a shared library"? I don't think so, but this ordering makes the wording clearer.

Reply:
That works.